base-notebook/test/test_container_options.py

```diff
@@ -77,7 +77,6 @@ def test_sudo_path(container):
     c = container.run(
         tty=True,
         user='root',
-        environment=['GRANT_SUDO=yes'],
```

Comment:
It'd be nice to keep the `GRANT_SUDO` option in this test to check that `jovyan` can run the `sudo which jupyter` command, and write another test (or parameterize this one) to check the case where the container launches as root without granting the user sudo.